setup.py

```diff
@@ -8,20 +8,18 @@
 import sys
 from setuptools import setup
 
-VERSION = '0.9.0'
+VERSION = '0.9.1'
 
 DEPENDENCIES = [
     'argcomplete',
     'jmespath',
     'pygments',
     'pyyaml',
-    'tabulate'
+    'tabulate',
+    # On Windows, colorama is required for legacy terminals.
+    'colorama; sys_platform == "win32"'
```

Comment:
I am actually wondering which one is the best:

- `os_name`
- `sys_platform`
- `platform_system`

`portalocker` is one dependency of Azure CLI and it uses:

https://github.com/WoLpH/portalocker/blob/00cffcb1831208691d3bf6850df4768989a0bd4c/setup.py#L133

```py
            'pywin32>=226; platform_system == "Windows"',
```

Reply:
> Would ever be the case `sys_platform` would be something else than `win32` in a Windows? Like `win64`?

I don't think so. I am running 64-bit Python on 64-bit Windows:

```
> python -c "import sys; print(sys.platform)"
win32
```

https://docs.python.org/3/library/sys.html#sys.platform only mentions `win32` too.